scanner/src/main/kotlin/scanners/ScanCode.kt

```diff
@@ -118,7 +118,7 @@ class ScanCode(
         }
     }
 
-    override val scannerVersion = "3.2.1-rc2"
+    override val expectedVersion = "3.2.1-rc2"
```

Comment:
out-of-scope: Would it make sense to ditch the backwards compatibility code / tests of ScanCode? Haven't checked actually whether they still exists, but recall seeing them not too long ago.

Reply:
It probably makes sense to do that in the near future, yes. Maybe the advent of ScanCode 3.2 *final* would be a good opportunity to remove that code.